integrations/slack

```diff
@@ -109,8 +109,8 @@ if __name__ == "__main__":
         # Read arguments
         bad_arguments = False
         if len(sys.argv) >= 2:
```

Comment:
If we run the integration with a missing parameter, the script raises an IndexError exception.

```
# integrations/slack logs/alerts/alerts.json apikey hookurl
Traceback (most recent call last):
  File "integrations/slack", line 112, in <module>
    msg = '{0} {1} {2} {3} {4}'.format(now, sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4])
IndexError: list index out of range
```